js/package.json

```diff
@@ -25,8 +25,8 @@
     "webpack": "^3.5.5"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.0.0",
-    "@jupyter-widgets/controls": "^1.0.0",
+    "@jupyter-widgets/base": "^2.0.1",
```

Comment:
Since we still want to support classic notebooks, my reading of https://github.com/jupyter-widgets/ipywidgets/pull/2472 and https://github.com/jupyter-widgets/ipywidgets/blob/master/docs/source/changelog.md#75 is that this should change to:

```
 "@jupyter-widgets/base": "^1.1 || ^2"
```